### PR TITLE
Table component for Collibra data

### PR DESCRIPTION
**Pull request by wenche**
Solve #[AB#82489](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/82489)

Doesn't take formatted text inside the table cells into account (yet, will or will not be prioritised in another user story)

### CODE REVIEW

**Review by nilsml — APPROVED** [review]
LGTM

File: components/AssetTabContent/OverviewContent.tsx [hunk 1]
```diff
@@ -31,8 +31,9 @@ const OverviewSubTitle: FunctionComponent<PropsWithChildren> = ({ children }) =>
 )
 
 export const OverviewContent = ({ content }: Props) => {
-  const usePortableText = config.USE_PORTABLE_TEXT
-
+  const usePortableText = process.env.NEXT_PUBLIC_USE_PORTABLE_TEXT === "true"
```
Comment (aronhoyer) [inline components/AssetTabContent/OverviewContent.tsx:34]: why was this removed from `config.ts`?

File: htmlParsing/rules.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { v4 as uuidv4 } from "uuid"
```
Comment (aronhoyer) [inline htmlParsing/rules.ts:1]: node's built-in crypto package has a function to generate a v4 uuid. see https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

File: components/AssetTabContent/OverviewContent.tsx (hunk 1 above)
Comment (wenche) [reply components/AssetTabContent/OverviewContent.tsx:34]: I left a comment about that one, it didn't update properly 🤷‍♀️  Was left with false server side and true client side, even after restarting the server ++

File: htmlParsing/rules.ts (hunk 2 above)
Comment (wenche) [reply htmlParsing/rules.ts]: Sweet, didn't know :)

File: components/AssetTabContent/OverviewContent.tsx (hunk 1 above)
Comment (aronhoyer) [reply components/AssetTabContent/OverviewContent.tsx:34]: that's peculiar 🤔

**Review by aronhoyer — APPROVED** [review]
